From 51e2d61e94374072e50b5668b9f34f46d45969a1 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 8 May 2014 14:37:08 -0400
Subject: [misc] mei: client.h cleanup

Message-id: <1399559828-6449-1-git-send-email-prarit@redhat.com>
Patchwork-id: 79565
O-Subject: [RHEL7.1 PATCH BZ 1094732 01/76] mei, client.h cleanup [v2]
Bugzilla: 1088953
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088953

RHEL7 commit 2002c42f2dc47a38bb1b34c4494190cf64f04061, linux-3.10.y stable
backport of e2b31644e999e8bfe3efce880fb32840299abf41, places the backported
function mei_cl_is_transitioning() in the wrong place in
drivers/misc/mei/client.h.

This patch puts the function in the correct place so that future backports
work.

RHEL_only.

[v2]: fix spacing issue

Cc: Dean Nelson <dnelson@redhat.com>
Cc: Stefan Assmann <sassmann@redhat.com>
Cc: Tony Camuso <tcamuso@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/mei/client.h b/drivers/misc/mei/client.h
index a2b70e6..b2898f4 100644
--- a/drivers/misc/mei/client.h
+++ b/drivers/misc/mei/client.h
@@ -76,13 +76,6 @@ static inline bool mei_cl_cmp_id(const struct mei_cl *cl1,
   (cl1->host_client_id == cl2->host_client_id) &&
   (cl1->me_client_id == cl2->me_client_id);
 }
-static inline bool mei_cl_is_transitioning(struct mei_cl *cl)
-{
- return (MEI_FILE_INITIALIZING == cl->state ||
-  MEI_FILE_DISCONNECTED == cl->state ||
-  MEI_FILE_DISCONNECTING == cl->state);
-}
-
 
 int mei_cl_flow_ctrl_creds(struct mei_cl *cl);
 
@@ -96,6 +89,12 @@ static inline bool mei_cl_is_connected(struct mei_cl *cl)
   cl->dev->dev_state == MEI_DEV_ENABLED &&
   cl->state == MEI_FILE_CONNECTED);
 }
+static inline bool mei_cl_is_transitioning(struct mei_cl *cl)
+{
+ return (MEI_FILE_INITIALIZING == cl->state ||
+  MEI_FILE_DISCONNECTED == cl->state ||
+  MEI_FILE_DISCONNECTING == cl->state);
+}
 
 bool mei_cl_is_other_connecting(struct mei_cl *cl);
 int mei_cl_disconnect(struct mei_cl *cl);
-- 
1.7.1